src/sentry/dynamic_sampling/prioritise_projects.py

```diff
@@ -0,0 +1,92 @@
+import logging
+import time
+from collections import defaultdict
+from datetime import datetime, timedelta
+from typing import Mapping, Sequence, Tuple
+
+from snuba_sdk import (
+    Column,
+    Condition,
+    Direction,
+    Entity,
+    Function,
+    Granularity,
+    Op,
+    OrderBy,
+    Query,
+    Request,
+)
+
+from sentry.dynamic_sampling.rules.utils import OrganizationId, ProjectId
+from sentry.sentry_metrics.indexer.strings import TRANSACTION_METRICS_NAMES
```

Comment:
As we discussed earlier, we don't want this to be loaded in directly. It should be accessed by a new method on Indexer::Base (something like `get_org_agnostic_strings(Sequence[str])-> Mapping[str,int]`)

Reply:
@RaduW is it anything related to you PR?

Reply:
@onewland done, PTAL